framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
In other words: the PR makes disabling a Widget and a Command different things. Disabling the Command disabled just the Command. Disabling the Buttons, disables the button and takes care it cannot be executed by a Command when beeing in this state.

That's at least not as broken as it is now.